enclave-tls/rules/wolfssl_env.mk

```diff
@@ -42,7 +42,7 @@ Wolfssl_Cflags := \
   -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_ALWAYS_VERIFY_CB -DKEEP_PEER_CERT -Wno-stringop-truncation \
   -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_KEY_GEN -DWOLFSSL_CERT_GEN \
   -DWOLFSSL_CERT_EXT -DWOLFSSL_ALWAYS_VERIFY_CB -DKEEP_PEER_CERT \
-  -DWOLFSSL_TEST_CERT -DWOLFSSL_SMALL_CERT_VERIFY
+  -DWOLFSSL_TEST_CERT -DWOLFSSL_SMALL_CERT_VERIFY -DHAVE_TM_TYPE
```

Comment:
It looks like this macro is used to disable USE_WOLF_TM which prevents from using `struct tm` defined by wolfssl?

Reply:
"tm" is defined in time.h in tlic as well. So it will show duplicating definition error. So I have to remove struct tm by the MACRO

Reply:
Please add this description as the comment.